tools/inliner.c

```diff
@@ -167,7 +167,7 @@ int main(int argc, char *argv[])
      *   __c("}\n");
      */
     write_str("void __c(char *src) {\n");
-    write_str("    strbuf_puts(SOURCE, src);\n");
+    write_str("    dynarr_extend(SOURCE, src, strlen(src));\n");
```

Comment:
It is weird to invoke `dynarr_extend` here. Conceptually, it is used for code generation, implying put/generate/output.